.github/workflows/rubocop.yml

```diff
@@ -6,8 +6,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: RuboCop Linter Action
-      uses: andrewmcodes/rubocop-linter-action@v3.3.0
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.1 
```

Comment:
I recently [updated](https://github.com/cyril/accept_language.rb/commit/b9ab94d9fb1ef1774fbbcba4927eb0d543ae1610) Ruby to version 3.2. Could you update this line? Then we'll merge 👍

Reply:
@cyril This should be fixed.